packages/fury-adapter-swagger/lib/parser.js

```diff
@@ -366,6 +366,45 @@ class Parser {
           });
         }
 
+        if (this.swagger.info.contact) {
+          this.withPath('contact', () => {
+            const { Link } = this.namespace.elements;
+            const { name, url, email } = this.swagger.info.contact;
+
+            if (url) {
+              const link = new Link();
+              link.relation = 'contact';
+              link.href = url;
+
+              if (name) {
+                link.title = name;
+              }
+
+              if (this.generateSourceMap) {
+                this.createSourceMap(link, this.path);
```

Comment:
```js
if (this.generateSourceMap) {
  this.withPath('url', () => {
    this.createSourceMap(link.href, this.path);
  });
}
```

We should apply the source map to the StringElement representing the HREF with the source map to the URL instead of the source map on the Link element

Reply:
change on next commit

Reply:
change on next commit, also for email